packages/schema/src/config/experimental.ts

```diff
@@ -79,11 +79,10 @@ export default defineUntypedSchema({
      */
     payloadExtraction: {
       async $resolve(enabled, get) {
-        enabled = enabled ?? false
-        if (enabled) {
+        if (enabled == null) {
```

Comment:
I prefer to make it explicit:

```suggestion
        if (enabled === null || enabled === undefined) {
```